src/Stormpath/Authc/Api/ApiAuthenticationResult.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Stormpath\Authc\Api;
+
+
+class ApiAuthenticationResult extends AuthenticatorResult
+{
+
+}
```

Comment:
@ecrisostomo actually, the spec shows different output for each type of call.  Check out the spec @omgitstom is showing here https://github.com/stormpath/stormpath-sdk-spec/blob/master/specifications/api_auth.md